recruiter/tests.py

```diff
@@ -276,3 +284,14 @@ def test_update_recruiter_account_settings_with_valid_data(example_recruiter, ex
     assert example_recruiter_from_db.phone_number == new_phone_number
     # Test that untouched data hasn't changed
     assert example_recruiter_from_db.company == example_recruiter.company
+
+
+@pytest.mark.django_db
+def test_add_new_comapny(valid_company_data, client):
+    response = client.post("/companyRegister/", data=valid_company_data)
+    assert response.status_code == 302
+    company = Company.objects.get(name=valid_company_data["name"])
+    # Test that the new data was updated in the DB
+    assert company.name == valid_company_data["name"]
+    assert company.description == valid_company_data["description"]
+    assert company.website_url == valid_company_data["website_url"]
```

Comment:
Where are the usual form tests (for basic display and validation?)

Reply:
will be fixed.

Reply:
still missing validation test.

Reply:
I search how can I test the template HTML after redirect and not after render. 
so in this case assertTemplateUsed is not helping, and also assertRedirects because that gets as input a URL address and not an HTML template.

Reply:
what do mean by validation test?

Reply:
@ifireball did you mean to this check?
`assert response.url == "/company_created_successfully"`

Reply:
I mean passing in bad data to the form and making sure errors are displayed as needed. All you other forms have a test like this

Reply:
ok just added invalid data tests.

Reply:
@ifireball  is that ok?